tests/cross_cluster_live_migration/conftest.py

```diff
@@ -1,39 +1,519 @@
+def remote_cluster_enabled_feature_gate_and_configured_hco_live_migration_network(
+    remote_cluster_hyperconverged_resource_scope_package,
+    remote_admin_client,
+    remote_cluster_network_for_live_migration,
+    remote_cluster_hco_namespace,
+):
+    """
+    Configure the live migration network for HyperConverged resource on the remote cluster.
+    """
+    virt_handler_daemonset = get_daemonset_by_name(
+        admin_client=remote_admin_client,
+        daemonset_name=VIRT_HANDLER,
+        namespace_name=remote_cluster_hco_namespace.name,
+    )
+
+    with ResourceEditorValidateHCOReconcile(
+        patches={
+            remote_cluster_hyperconverged_resource_scope_package: {
+                "spec": {
+                    "featureGates": {"decentralizedLiveMigration": True},
+                    "liveMigrationConfig": {"network": remote_cluster_network_for_live_migration.name},
+                }
+            }
+        },
+        list_resource_reconcile=[KubeVirt],
+        wait_for_reconcile_post_update=True,
+        admin_client=remote_admin_client,
+    ):
+        wait_for_virt_handler_pods_network_updated(
+            client=remote_admin_client,
+            namespace=remote_cluster_hco_namespace,
+            network_name=remote_cluster_network_for_live_migration.name,
+            virt_handler_daemonset=virt_handler_daemonset,
+        )
+        yield
+
+    wait_for_virt_handler_pods_network_updated(
+        client=remote_admin_client,
+        namespace=remote_cluster_hco_namespace,
+        network_name=remote_cluster_network_for_live_migration.name,
+        virt_handler_daemonset=virt_handler_daemonset,
+        migration_network=False,
+    )
+
+
+@pytest.fixture(scope="package")
+def mtv_namespace(admin_client):
+    return Namespace(name="openshift-mtv", client=admin_client, ensure_exists=True)
+
+
+@pytest.fixture(scope="package")
+def forklift_controller_resource_scope_package(admin_client, mtv_namespace):
+    return ForkliftController(
+        name="forklift-controller", namespace=mtv_namespace.name, client=admin_client, ensure_exists=True
+    )
+
+
+@pytest.fixture(scope="package")
+def local_cluster_enabled_mtv_feature_gate_ocp_live_migration(forklift_controller_resource_scope_package):
+    forklift_spec_dict = deepcopy(forklift_controller_resource_scope_package.instance.to_dict()["spec"])
+    forklift_spec_dict["feature_ocp_live_migration"] = "true"
+    with ResourceEditor(patches={forklift_controller_resource_scope_package: {"spec": forklift_spec_dict}}):
+        yield
+
+
+@pytest.fixture(scope="module")
+def mtv_forklift_services_route_host(admin_client, mtv_namespace):
+    """
+    Get the forklift-services route host.
+    """
+    forklift_services_route = Route(
+        client=admin_client,
+        name="forklift-services",
+        namespace=mtv_namespace.name,
+        ensure_exists=True,
+    )
+    route_host = forklift_services_route.instance.spec.host
+    assert route_host
```

Comment:
for easier debugging please add an assert message which gives context to the error